screens/ChatRoom/index.tsx

```diff
@@ -442,6 +454,21 @@ const ChatRoom = ({navigation, route}: ChatRoom) => {
 
   useEffect(() => {
     async function callApi() {
+      if (
+        chatroomDetails?.chatroom.type == 10 &&
```

Comment:
we are use `chatroomDetails?.chatroom.type` many times store it in a variable and then use this. Same for `chatroomDetails?.chatroom?.id`, `chatroomDetails?.chatroom?.follow_status` make variables for each of these values and then use it in the code.